tests/lib/test_Repo_config.py

```diff
@@ -12,84 +14,87 @@
 #
 # 'system` and 'global' are left untested, to not tamper with developer machines.
 
-variants = {
-    'default': {
+
+@params({
+    'default': {"variant": {
         'args': ('onyo.test.set-config', 'default'),
-        'file': '.onyo/config',
+        'file': '.onyo/config'},
     },
-    'local': {
+    'local': {"variant": {
```

Comment:
Not so much a nit as rather a note for the future: in this case, `variant` isn't needed. Instead, `args` and `file` can be each passed to the test function.

No need to change it here. Just saying. :-)